src/NossoMercadoLivreAPI.Domain/Request/ProfileRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+using NossoMercadoLivreAPI.Domain.Request.Base;
+
+namespace NossoMercadoLivreAPI.Domain.Request
+{
+    public class ProfileRequest : BaseRequest
```

Comment:
*Avaliamos de forma lógica o nível de complicação de cada trecho de código. Aqui não tem espaço para feeling. Usamos a teoria da carga cognitiva e medimos o nível de pontos de entendimentos necessário por trecho*

Um objeto de request é super específico, não precisamos de generalização. Toda generalização deixa o código mais complicado e devemos fazer só quando for estritamente necessário, o que não é o caso aqui.